templates/CRM/Contribute/Form/Contribution/Main.tpl

```diff
@@ -321,7 +324,7 @@
       var isRecur = cj('input[id="is_recur"]:checked');
 
       var quickConfig = {/literal}'{$quickConfig}'{literal};
-      if (cj("#auto_renew") && quickConfig) {
+      if (cj("#auto_renew").length && quickConfig) {
```

Comment:
@seamuslee001 did your patch address this part of it by ensuring auto_renew is assigned?

Reply:
@eileenmcnaughton I don't think so